Tutorials/Level1Tutorial.md

```diff
@@ -0,0 +1,780 @@
+# Level 1 Tutorial
+
+## Table of Contents
+1. [What is SST](#WhatIsSST)
+2. [Components of SST](#ComponentsOfSST)
+3. [Basic Installation](#BasicInstallation)
+4. [Basic Commands](#BasicCommands)
+5. [Basic Runtime Functions](#BasicRuntimeFunctions)
+6. [Collecting Data from SST](#CollectingData)
+7. [Links to Documentation](#LinksToDocumentation)
+
+## What is SST: <a name="WhatIsSST"></a>
+  To combat the ever-changing and multifaceted technologies of the HPC 
```

Comment:
First use of "HPC" should be expanded as "High Performance Computing"

Reply:
resolved in a48b6c9